src/edu/ucsb/nceas/metacat/restservice/D1RestServlet.java

```diff
@@ -40,7 +44,7 @@
  */
 public class D1RestServlet extends HttpServlet {
 
-    protected Logger logMetacat;
+    protected Log logMetacat;
```

Comment:
```suggestion
    protected Log logMetacat = LogFactory.getLog(this.getClass());
```

Reply:
See the commit 30d9be7163527d0d655d521e3266d0e0a187ce3.